doc/source/examples/metrology/three_receiver_cal/data/simulation.s2p

```diff
@@ -1,1618 +1,1618 @@
-
-89.475           0.0831973271685238 -66.1507465666787 0.989024949919211 -112.250792151651 0.989024949919214 -112.250792151651 0.0820141519464012 21.5250199699923 
-! Gamma         !               0.280886121641811 1577.86398659652 0.280885082750083 1577.86593683708 
-! Port Impedance275.997702490515 0.0977300693684183 276.019632703776 0.0978986437676333 
-
-89.55            0.0829970470984323 -66.3242244017108 0.989043832565401 -115.14426642367 0.989043832565406 -115.14426642367 0.0818168504511698 15.9115097385338 
-! Gamma         !               0.28080058760216 1579.73206041133 0.280799553057184 1579.73400834827 
-! Port Impedance275.90229557344  0.0977299919785913 275.924217392056 0.0978987217885426 
-
-89.625           0.0827953847660856 -66.4976270663726 0.989062786094827 -118.03685302204 0.989062786094831 -118.03685302204 0.0816181639542071 10.2998587337155 
-! Gamma         !               0.280715517660658 1581.59949016452 0.280714487449462 1581.6014358041  
-! Port Impedance275.807226142358 0.0977303097076067 275.829139595978 0.0978991950908056 
-
-89.7             0.0825923348598775 -66.6709477298723 0.989081811326553 -120.928557101572 0.989081811326556 -120.928557101572 0.0814180858959231 4.69006528525823 
-! Gamma         !               0.280630909550731 1583.46627813468 0.280629883660409 1583.46822148309 
-! Port Impedance275.71249249042  0.0977310202593581 275.734397608542 0.09790006137779 
-
-89.775           0.0823878922086671 -66.8441792389995 0.989100909119843 -123.819383806718 0.989100909119846 -123.819383806718 0.0812166098077885 -0.917871926735089 
-! Gamma         !               0.280546761020393 1585.33242658885 0.280545739438103 1585.33436765228 
-! Port Impedance275.618092922182 0.0977321213562437 275.639989734152 0.0979013183713715 
-
-89.85            0.0821820517967471 -67.0173141144148 0.98912008037424 -126.709338270958 0.989120080374243 -126.709338270958 0.0810137293273589 -6.52395386694479 
-! Gamma         !               0.280463069832124 1587.19793778265 0.280462052545088 1587.19987656724 
-! Port Impedance275.52402575351  0.0977336107389831 275.54591428852  0.0979029638117508 
-
-89.925           0.0819748087788862 -67.1903445471563 0.989139326029622 -129.598425616181 0.989139326029625 -129.598425616181 0.0808094382134603 -12.1281811401887 
-! Gamma         !               0.280379833762757 1589.06281396033 0.280378820758261 1589.06475047222 
-! Port Impedance275.430289311483 0.0977354861664361 275.452169598573 0.0979049954572732 
-
-90               0.0817661584954339 -67.3632623953684 0.989158647066246 -132.486650952035 0.989158647066249 -132.486650952035 0.0806037303615188 -17.730553986953 
-! Gamma         !               0.280297050603356 1590.92705735488 0.28029604186875 1590.92899160015 
-! Port Impedance275.336881934298 0.097737745415425 275.358754002361 0.0979074110842495 
-
+! Touchstone file from project Project26
```

Comment:
Would be better to revert unmodified files to their original state to avoid clutter in the PR

Reply:
Reverted all files that were only different in how they handled line feeds (i.e. CRLF vs LF)